libs/agora/gene-comparison-tool/src/lib/gene-comparison-tool.component.html

```diff
@@ -9,43 +9,30 @@
       <div class="gct-header-inner">
         <div class="gct-header-left">
           <p-button
+            label="Filter Genes"
             (click)="filterPanel.toggle()"
+            [rounded]="true"
+            icon="pi pi-filter"
```

Comment:
The current dev site uses a solid filter icon for this button -- 

<img width="220" alt="image" src="https://github.com/user-attachments/assets/15271e7e-6a6e-48e7-974d-fbac699f1aaf" />

Should we use `pi-filter-fill` instead? 

```suggestion
            icon="pi pi-filter-fill"
```